sdk/src/main/java/com/mapbox/maps/renderer/gl/GlUtils.kt

```diff
@@ -0,0 +1,70 @@
+package com.mapbox.maps.renderer.gl
+
+import android.opengl.GLES20
+import android.opengl.Matrix
+import com.mapbox.maps.BuildConfig
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+import java.nio.FloatBuffer
+
+internal object GlUtils {
+
+  fun FloatArray.put(vararg values: Float) {
+    values.forEachIndexed { index, value ->
```

Comment:
nit:
should you check length of `values`?

Reply:
Well, it's complicated. I'd say no since these arrays / buffers represent matrices for GL and should be of size 16 always

Reply:
it was just a question. 
because looks like common util function. maybe you want to reuse it later